src/app.vue

```diff
@@ -3,24 +3,98 @@
     <nav>
       <div class="inner">
         <router-link class='logo' to='/'>
-          <img src="@/assets/header-logo.svg" alt=""/>
-          <span>explorer</span>
+          <img src="@/assets/header-logo.svg" alt="" />
+          <span class="app-name">explorer</span>
         </router-link>
         <div class="links">
           <router-link class='link' to='/'>
             Dashboard
           </router-link>
-          <router-link class='link'  to='/blocks'>
+          <router-link class='link' to='/blocks'>
             Blocks
           </router-link>
           <router-link class='link' to='/tx'>
             Transactions
           </router-link>
         </div>
+        <div class="mobile-nav">
+          <div class="pageName">
+            <strong v-if="$route.name === 'Index'">Dashboard</strong>
+            <strong v-if="$route.name === 'BlockList'">Blocks</strong>
+            <strong v-if="$route.name === 'Transaction'">Transactions</strong>
+          </div>
+          <div class="burger" @click='expandMenu'>
+            <img src="@/assets/burger.svg" alt="" />
+          </div>
+          <div class="mobile-nav-fixed" :class="{ active: isOpened}" @click="expandMenu">
```

Comment:
Instead of `.active` I propose to use `v-if` and [`transition`](https://vuejs.org/v2/guide/transitions.html) component.